android/src/com/frostwire/android/gui/activities/MainActivity.java

```diff
@@ -704,28 +605,21 @@ private void onShutdownDialogButtonPositive() {
         Offers.showInterstitial(this, Offers.PLACEMENT_INTERSTITIAL_EXIT, true, false);
     }
 
-    private void syncSlideMenu() {
+    public void syncNavigationMenu() {
         invalidateOptionsMenu();
-        navView.setCheckedItem(getNavMenuIdByFragment(getCurrentFragment()));
-    }
-
-    private void refreshMenuRemoveAdsItem() {
-        // only visible for basic or debug build and if ads have not been disabled.
-        int visibility = ((Constants.IS_GOOGLE_PLAY_DISTRIBUTION || Constants.IS_BASIC_AND_DEBUG) && !Offers.disabledAds()) ?
-                View.VISIBLE :
-                View.GONE;
-        menuRemoveAdsItem.setVisibility(visibility);
+        navigationMenu.updateCheckedItem(getNavMenuIdByFragment(getCurrentFragment()));
     }
 
     private void setupFragments() {
         search = (SearchFragment) getFragmentManager().findFragmentById(R.id.activity_main_fragment_search);
         library = (BrowsePeerFragment) getFragmentManager().findFragmentById(R.id.activity_main_fragment_browse_peer);
         transfers = (TransfersFragment) getFragmentManager().findFragmentById(R.id.activity_main_fragment_transfers);
-        hideFragments(getFragmentManager().beginTransaction()).commit();
     }
 
-    private FragmentTransaction hideFragments(FragmentTransaction ts) {
-        return ts.hide(search).hide(library).hide(transfers);
+    private void hideFragments() {
+        FragmentTransaction tx = getFragmentManager().beginTransaction();
```

Comment:
refactor to silence warning on transaction possibly not commited.
Also cleaner when `hideFragments()` is used.